libcudacxx/include/cuda/std/__complex/nvbf16.h

```diff
@@ -161,9 +161,9 @@ class _CCCL_TYPE_VISIBILITY_DEFAULT _CCCL_ALIGNAS(alignof(__nv_bfloat162)) compl
     return *this;
   }
 
-  _CCCL_HOST operator ::std::complex<value_type>() const
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_CONSTEXPR_STD_COMPLEX_ACCESS operator ::std::complex<value_type>() const
```

Comment:
We cannot mark this as `_LIBCUDACXX_HIDE_FROM_ABI` because that includes `__host__ __device__` only this would be possible 
```suggestion
  _CCCL_HIDE_FROM_ABI _LIBCUDACXX_CONSTEXPR_STD_COMPLEX_ACCESS operator ::std::complex<value_type>() const
```